routing/http/client/transport.go

```diff
@@ -36,3 +41,35 @@ func (l *limitReadCloser) Read(p []byte) (int, error) {
 	}
 	return n, err
 }
+
+// ImportPath is the canonical import path that allows us to identify
+// official client builds vs modified forks, and use that info in User-Agent header.
+const ImportPath = "github.com/ipfs/go-libipfs/routing/http/client"
+
+// moduleVersion returns a useful user agent version string allowing us to
+// identify requests coming from officialxl releases of this module.
+func moduleVersion() string {
```

Comment:
Can we add a test for this?  Really I'm interested in making sure a test fails if this code is moved and `ImportPath` is not updated (or use reflection or something to get the import path dynamically).

Reply:
@guseggert simplified and added test in https://github.com/ipfs/go-libipfs/pull/31/commits/4e11e8b46555c5f623eec39fb25f5d6c56f4ea08